testing-resources/include/aws/external/gtest.h

```diff
@@ -2500,7 +2500,7 @@ class MutexBase {
 // This allows initialization to work whether pthread_t is a scalar or struct.
 // The flag -Wmissing-field-initializers must not be specified for this to work.
 # define GTEST_DEFINE_STATIC_MUTEX_(mutex) \
-    ::testing::internal::MutexBase mutex = { PTHREAD_MUTEX_INITIALIZER, false }
+    ::testing::internal::MutexBase mutex = {PTHREAD_MUTEX_INITIALIZER, false, 0}
```

Comment:
This is google's code, and I don't plan on changing it. Eventually we'll swap out the amalgomated source and this change will be lost anyways.